cmake/FindGROMACS.cmake

```diff
@@ -0,0 +1,22 @@
+# - Find GROMACS
```

Comment:
Gromacs comes with cmake config files, so there is no need to write our own `FindGROMACS.cmake`, just use:
```
find_package(GROMACS CONFIG)
```